pb/sensorsinfo_proto/sensorsinfo.proto

```diff
@@ -76,6 +76,9 @@ message Sensor{
     int32 Reserved = 9;
     bool Batch = 10;
     bool WakeUp = 11;
+    bool isActive = 12;
+    double runningSamplingRateHz = 13;
```

Comment:
Follow same format for names - 1st letter is capital. 

Also instead of running, use Active or Actual

Reply:
Has running been used in other parts of the code?

Reply:
These value will be used in the js/bars.js file to show the information in a floating window.